grpc/src/main/java/com/replica/RouterServer.java

```diff
@@ -177,6 +194,34 @@ private void start() throws Exception {
 
     }
 
+    // inspired by https://stackoverflow.com/questions/32933367/monitoring-the-size-of-the-netty-event-loop-queues and
+    // https://github.com/mpusher/mpush/blob/f8d5c97f30b2bde12f1b60d11709ae1b75587a2e/mpush-tools/src/main/java/com/mpush/tools/Utils.java#L213
+    public static void recordNetworkingMetrics(final StatsDClient statsDClient, final EventLoopGroup elg, final String componentName) {
+        logger.info("Recording networking metrics");
+        int index = 0;
+        int poolSize = 0, queueSize = 0, activeCount = 0;
+        for (final EventExecutor eventExecutor : elg) {
+            poolSize++;
+            if (eventExecutor instanceof SingleThreadEventExecutor) {
+                final SingleThreadEventExecutor singleExecutor = (SingleThreadEventExecutor) eventExecutor;
+                String metricName = "EventLoopGroup_" + componentName +  "_pending_tasks";
+                int pendingTasks = singleExecutor.pendingTasks();
+                statsDClient.gauge(metricName, pendingTasks, Integer.toString(index));
+                index++;
+
+                queueSize += pendingTasks;
+                ThreadProperties tp = singleExecutor.threadProperties();
+                if (tp.state() == Thread.State.RUNNABLE) {
+                    activeCount++;
+                }
+            }
+        }
+        logger.info(componentName + " pool size: " + poolSize + " ; queue size: " + queueSize + " ; active count: " + activeCount);
```

Comment:
Good call - I decided to just comment out the infra block that sets up this logging thread (and calls `recordNetworkingMetrics`), while keeping the function around in case we want to investigate this more in the future. From what we saw with the logs this was spitting out, GRPC queue sizes didn't seem to point to any obvious issues that needed addressing, so I'm OK with turning it off for now (but available quickly in the future if we need that visibility again at any point)